From: Mike Engel <Mike.Engel@digi.com>
Date: Wed, 23 Jun 2021 14:55:14 +0200
Subject: [PATCH 1/2] library: add swupdate static API library

Signed-off-by: Mike Engel <Mike.Engel@digi.com>
---
 library/cloudconnector.pc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library/cloudconnector.pc b/library/cloudconnector.pc
index 5cb280c..be8ef80 100644
--- a/library/cloudconnector.pc
+++ b/library/cloudconnector.pc
@@ -8,6 +8,6 @@ Description: Cloud Connector library
 Version:1.0
 
 Requires.private: libconfuse openssl recovery zlib
-Libs: -L${libdir} -lcloudconnector
+Libs: -L${libdir} -lcloudconnector -lswupdate
 Libs.private: -lpthread
 Cflags: -I${includedir}/cloudconnector -I${includedir}
-- 
